packages/components/test/number.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import "@testing-library/jest-dom";
+import { SquiggleChart } from "../src/index";
+
+test("Precision", async () => {
+  const { container: highPrecision } = render(
+    <SquiggleChart code={"1.25476"} numberPrecision={5} />
```

Comment:
Minor note: you don't have to put `{}` around constant strings, `code="1.25476"` is enough and has less visual noice.

(this is almost too tiny to point out, but I've noticed this before; I wish prettier would auto-format it)